addons/web_tour/static/src/js/tip.js

```diff
@@ -161,6 +185,15 @@ var Tip = Widget.extend({
             collision: "none",
         });
 
+        console.log('State : ', this.state);
+        let $parent = this.$el.parent();
+        let paddingTop = parseInt(this.$el.parent().css('padding-top').replace('px', ''));
+        if (this.state === 'up') {
+            this.$el.css({position: 'fixed', top: $parent.offset().top + paddingTop + this.$el.innerHeight()});
+        } else if (this.state === 'down') {
+            this.$el.css({position: 'fixed', top: $parent.offset().top + $parent.innerHeight() - this.$el.innerHeight() * 2});
+        }
```

Comment:
I don't think the 'position' should ever change, in fact if the left panel was not full height I don't think this would work. Could you clean your PR so that it is easier to review? :) Remove useless logs, split your commit, etc.

In any case, it feels like there is too much code for what we are trying to achieve.

What we have before your task:
1. When the tip is closed, it is in the ancestor-which-scrolls of the anchor element (get_ideal_location), so that we don't it to reposition it at each scroll, it is automatically done by the browser.
2. When the tip is opened, it is in the body to be over everything else and if the user scroll, it stays where it is.

After your work, those two points should still be true. We don't want to reposition the tip on each scroll, ever. What we want is a third point:
3. When the ancestor-which-scrolls of the anchor element (thus the parent of the tip, see (1)) scrolls, then we check if the anchor is visible or not. If not, the tip has to point at a different anchor: the boundaries of the ancestor-which-scrolls to indicate to the user that he has to scroll. Like any other tip, the tip has thus to be moved in the ideal location: the ancestor-which-scrolls of the ancestor-which scrolls (probably the body in this case but you can't be sure, you have to find it with the get_ideal_location function). Once it is at the right location, then you simply have to reposition it.

I think that this is what you have done for the most part, but instead of moving the tip you make it position: fixed which is not right. Try to reorganize your work and implement what I explained here.